lib/capistrano/tasks/database.rake

```diff
@@ -0,0 +1,109 @@
+namespace :forkcms do
+  namespace :database do
+
+    backup_file = 'mysql_backup.sql'
+
+    desc 'Create a backup of the database'
+    task :backup do
+      on roles(:web) do
+        # Fetch our flags to reach the database
+        flags = get_mysql_connection_flags
+
+        # Abort if no flags are found
+        if flags == nil
+          next
+        end
+
+        execute :mysqldump, "#{flags} --skip-lock-tables > #{release_path}/#{backup_file}"
+      end
+    end
+
+    desc 'Restore the database from a backup file'
+    task :restore do
+      on roles(:web) do
+        # Check if the file exists.
+        if capture("if [ -f #{release_path}/#{backup_file} ]; then echo 'yes'; fi").chomp != 'yes'
+          puts "No backup file found, create a backup first"
+          next
+        end
+
+        Rake::Task["forkcms:database:execute"].invoke("#{release_path}/#{backup_file}")
+      end
+    end
+
+    desc 'Execute an SQL file'
+    task :execute, [:file] do |t, arguments|
+      on roles(:web) do
+        # Stop if the file does not exist
+        if capture("if [ -f #{arguments[:file]} ]; then echo 'yes'; fi").chomp != 'yes'
+          puts "File at #{arguments[:file]} does not exist"
+          next        
```

Comment:
What is this `next`?

Reply:
`next` is like an early return in Rake, it moves to the next task. In reality though, it's better to abort everything so I removed it